### PR TITLE
[WWST-6480] First batch of Device Join Names change

### PR DESCRIPTION
**Pull request by PKacprowiczS**
as @tpmanley has proposed, I've divided DJN change into few batches. Here's first.
@greens @dkirker @KKlimczukS @MGoralczykS @ZWozniakS @MWierzbinskaS @BJanuszS

### CODE REVIEW

File: devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy [hunk 1]
```diff
@@ -198,7 +198,8 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 	if (cmd.notificationStatus == 0xFF) {
 		switch (cmd.event) {
 			case 0x09: //TAMPER
-				createEvent([name: "tamper", value: "detected"])
```
Comment (tpmanley) [inline devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy]: This shouldn't be part of this PR

File: devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy [hunk 2]
```diff
@@ -24,8 +24,7 @@ metadata {
         capability "Configuration"
         capability "Health Check"
 
-        fingerprint inClusters: "0000,0001,0003,000F,0020", outClusters: "0003,0019",
-                        manufacturer: "SmartThings", model: "tagv4", deviceJoinName: "Arrival Sensor"
+        fingerprint inClusters: "0000,0001,0003,000F,0020", outClusters: "0003,0019", manufacturer: "SmartThings", model: "tagv4", deviceJoinName: "Presence Sensor"
```
Comment (tpmanley) [inline devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy]: This should be held for the next PR until we know what the manufacturer name should be

File: devicetypes/smartthings/arrival-sensor.src/arrival-sensor.groovy [hunk 3]
```diff
@@ -23,9 +23,9 @@ metadata {
 		capability "Battery"
 		capability "Health Check"
 
-		fingerprint profileId: "FC01", deviceId: "019A"
-		fingerprint profileId: "FC01", deviceId: "0131", inClusters: "0000,0003", outClusters: "0003"
-		fingerprint profileId: "FC01", deviceId: "0131", inClusters: "0000", outClusters: "0006"
+		fingerprint profileId: "FC01", deviceId: "019A", deviceJoinName: "Presence Sensor"
```
Comment (tpmanley) [inline devicetypes/smartthings/arrival-sensor.src/arrival-sensor.groovy]: This should also be held for the next PR

File: devicetypes/smartthings/ct100-thermostat.src/ct100-thermostat.groovy [hunk 4]
```diff
@@ -20,9 +20,9 @@ metadata {
 		command "raiseCoolSetpoint"
 		command "poll"
 
-		fingerprint deviceId: "0x08", inClusters: "0x43,0x40,0x44,0x31,0x80,0x85,0x60"
-		fingerprint mfr:"0098", prod:"6401", model:"0107", deviceJoinName: "2Gig CT100 Programmable Thermostat"
-		fingerprint mfr:"0098", prod:"6501", model:"000C", deviceJoinName: "Radio Thermostat CT101"
+		fingerprint deviceId: "0x08", inClusters: "0x43,0x40,0x44,0x31,0x80,0x85,0x60", deviceJoinName: "Thermostat"
+		fingerprint mfr:"0098", prod:"6401", model:"0107", deviceJoinName: "2Gig Thermostat" //2Gig CT100 Programmable Thermostat
+		fingerprint mfr:"0098", prod:"6501", model:"000C", deviceJoinName: "Thermostat" //Radio Thermostat CT101
```
Comment (tpmanley) [inline devicetypes/smartthings/ct100-thermostat.src/ct100-thermostat.groovy:25]: Why doesn't this have a manufacturer?

Comment (PKacprowiczS) [reply devicetypes/smartthings/ct100-thermostat.src/ct100-thermostat.groovy:25]: I guess I've thought that Radio doesn't mean brand but, well, indicates being wireless... I don't know, back then it seemed to be reasonable...

Comment (tpmanley) [reply devicetypes/smartthings/ct100-thermostat.src/ct100-thermostat.groovy:25]: I think you're right. Sorry, I forgot we had discussed this before. I think it should be left as just "Thermostat".

File: devicetypes/fibargroup/fibaro-co-sensor-zw5.src/fibaro-co-sensor-zw5.groovy [hunk 5]
```diff
@@ -14,9 +14,9 @@ metadata {
 
 		attribute "coLevel", "number"
 
-		fingerprint mfr: "010F", prod: "1201", model: "1000"
-		fingerprint mfr: "010F", prod: "1201", model: "1001"
-		fingerprint mfr: "010F", prod: "1201"
+		fingerprint mfr: "010F", prod: "1201", model: "1000", deviceJoinName: "Fibaro Smoke Detector"
+		fingerprint mfr: "010F", prod: "1201", model: "1001", deviceJoinName: "Fibaro Smoke Detector"
+		fingerprint mfr: "010F", prod: "1201", deviceJoinName: "Fibaro Smoke Detector"
```
Comment (greens) [inline devicetypes/fibargroup/fibaro-co-sensor-zw5.src/fibaro-co-sensor-zw5.groovy]: repeating my question about calling CO sensors "Smoke Detector"s

File: devicetypes/juano2310/jawbone-user.src/jawbone-user.groovy [hunk 6]
```diff
@@ -4,7 +4,7 @@
  *  Author: juano23@gmail.com
  *  Date: 2013-08-15
  */
- // for the UI
+ //fortheUI
```
Comment (greens) [inline devicetypes/juano2310/jawbone-user.src/jawbone-user.groovy]: ?

Comment (PKacprowiczS) [reply devicetypes/juano2310/jawbone-user.src/jawbone-user.groovy]: I had bug somewhere in script that for some reason removes all spaces from couple of comments (afaik total 3 in whole repository, dunno why)...

File: devicetypes/smartthings/fortrezz-water-valve.src/fortrezz-water-valve.groovy [hunk 7]
```diff
@@ -19,8 +19,8 @@ metadata {
 		capability "Refresh"
 		capability "Sensor"
 
-		fingerprint deviceId: "0x1000", inClusters: "0x25,0x72,0x86,0x71,0x22,0x70"
-		fingerprint mfr:"0084", prod:"0213", model:"0215", deviceJoinName: "FortrezZ Water Valve"
+		fingerprint deviceId: "0x1000", inClusters: "0x25,0x72,0x86,0x71,0x22,0x70", deviceJoinName: "FortrezZ Valve"
```
Comment (tpmanley) [inline devicetypes/smartthings/fortrezz-water-valve.src/fortrezz-water-valve.groovy:22]: I think the manufacturer name should be left off here since it's a generic fingerprint.

File: devicetypes/smartthings/homeseer-multisensor.src/homeseer-multisensor.groovy [hunk 8]
```diff
@@ -20,7 +20,7 @@ metadata {
 		capability "Sensor"
 		capability "Battery"
 
-		fingerprint deviceId: "0x2101", inClusters: "0x60,0x31,0x70,0x84,0x85,0x80,0x72,0x77,0x86"
+		fingerprint deviceId: "0x2101", inClusters: "0x60,0x31,0x70,0x84,0x85,0x80,0x72,0x77,0x86", deviceJoinName: "HomeSeer Multipurpose Sensor"
```
Comment (tpmanley) [inline devicetypes/smartthings/homeseer-multisensor.src/homeseer-multisensor.groovy:23]: Since this is a generic fingerprint I think "HomeSeer" should be left out.

**Comment by tpmanley** [comment]
When the PRs are this large it makes it much easier to review if you use fixup commits with just the changes that were made so we don't have to review every line each time.

File: devicetypes/smartthings/fortrezz-water-valve.src/fortrezz-water-valve.groovy (hunk 7 above)
Comment (PKacprowiczS) [reply devicetypes/smartthings/fortrezz-water-valve.src/fortrezz-water-valve.groovy:22]: But the manufacturer name is still in the name of DTH, so it won't change anything in that area.

File: devicetypes/smartthings/homeseer-multisensor.src/homeseer-multisensor.groovy (hunk 8 above)
Comment (PKacprowiczS) [reply devicetypes/smartthings/homeseer-multisensor.src/homeseer-multisensor.groovy:23]: Brand name is also in DTH's name, so including DJN in this fingerprint has only device type naming unification purpose.

File: devicetypes/smartthings/fortrezz-water-valve.src/fortrezz-water-valve.groovy (hunk 7 above)
Comment (tpmanley) [reply devicetypes/smartthings/fortrezz-water-valve.src/fortrezz-water-valve.groovy:22]: OK, we can leave it. It does feel like something we can improve later since generic fingerprints could match any manufacturer's device.

**Comment by tpmanley** [comment]
@PKacprowiczS it looks like all the comments have been resolved. Can you squash/rebase the changes on the latest `master` branch? We can give it one last quick review today and then merge.

File: devicetypes/smartthings/fortrezz-water-valve.src/fortrezz-water-valve.groovy (hunk 7 above)
Comment (dkirker) [reply devicetypes/smartthings/fortrezz-water-valve.src/fortrezz-water-valve.groovy:22]: @tpmanley @PKacprowiczS  If the code should work with any device that matches that fingerprint, the `deviceJoinName` could be set to something generic, like just "Water Valve".

Comment (tpmanley) [reply devicetypes/smartthings/fortrezz-water-valve.src/fortrezz-water-valve.groovy:22]: I agree @dkirker but in the interest of getting this moving I think it's OK since the current behavior is to use the DTH name which had the manufacturer name in there already.

**Comment by PKacprowiczS** [comment]
@greens @tpmanley @dkirker please have a look at it, when you'll be able to

**Review by tpmanley — APPROVED** [review]

**Review by greens — APPROVED** [review]
